### PR TITLE
Doc site

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created landing page.
- created pages for all P0 components.

### CODE REVIEW

File: css/badge.css [hunk 1]
```diff
@@ -41,9 +41,10 @@
 
 .badge-container .avatar-xs ~ .status-badge {
   position: relative;
-  height: 0.4rem;
-  width: 0.4rem;
-  right: 0.7rem;
+  height: 0.5rem;
+  width: 0.5rem;
+  right: 0.5rem;
+  bottom: 0.1rem;
```
Comment (ritikaawasthy) [inline css/badge.css:47]: you px for smaller values

File: css/buttons.css [hunk 2]
```diff
@@ -4,11 +4,12 @@
   border: 0.5px solid var(--slate-300);
   width: max-content;
   height: max-content;
-  padding: 0.7rem 1rem;
+  padding: 0.7em 1em;
```
Comment (ritikaawasthy) [inline css/buttons.css:7]: would suggest using rem here

**Review by ritikaawasthy — APPROVED** [review]
do checkout the comments, all over looks great

File: css/buttons.css (hunk 2 above)
Comment (VedanthB) [reply css/buttons.css:7]: you've replaced it with `em`  but i think you meant to make it `rem`

File: pages/docs.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  text-underline-position: under;
+}
+
+.mySidenavToggle {
+  display: none;
+  position: fixed;
+}
+
+/* On screens that are 992px or less, */
+@media screen and (max-width: 992px) {
```
Comment (VedanthB) [inline pages/docs.css:68]: use `rem` here for max-width

File: pages/untiltled.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<!-- standard input -->
```
Comment (VedanthB) [inline pages/untiltled.html:1]: I think you didn't meant to commit this file

**Review by VedanthB — APPROVED** [review]
Good Job Ananya,
please keep your PRs small , it make its easier for others to give better to give reviews.
Everything looks good to go 🚀 

Another suggestion, you could use another font. (just a suggestion)

**Pull request merged**